src/Compilers/VisualBasic/Portable/Semantics/OverloadResolution.vb

```diff
@@ -915,13 +963,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
 
             Dim applicableNarrowingCandidateCount As Integer = 0
             Dim applicableInstanceCandidateCount As Integer = 0
+            Dim someCandidatesHaveOverloadResolutionPriority As Boolean = False
 
             ' First collect instance methods.
             If instanceCandidates.Count > 0 Then
 
+                someCandidatesHaveOverloadResolutionPriority = instanceCandidates.Any(Function(candidate) candidate.OverloadResolutionPriority <> 0)
```

Comment:
Specifically, I expect this filter to fail in the presence of an override.

Reply:
> Specifically, I expect this filter to fail in the presence of an override.

Given the way VB name lookup works, the least derived forms are also among the candidates. Therefore we get to them as well without explicitly traversing the overrides hierarchy.